testsuite/src/main/resources/convert/available/views/index.jsp

```diff
@@ -0,0 +1,18 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<!doctype html>
+<html>
+<head>
+    <title>Available converter test</title>
+</head>
+<body>
+    <h1>Available converter test</h1>
+
+    <div id="short">${short}</div>
```

Comment:
Actually, I'm a bit surprised that this works. Aren't `short`, `double`, etc. reserved keywords? And AFAIK such restrictions apply to EL as well?

Reply:
Hmm good point. I'll change it just to ensure we won't run into any weird behavior later. Thanks for pointing this out.

Reply:
Awesome! Not sure if this is really an issue TBH. It just came up in my head during the review.

Reply:
Done. Now we shouldn't have any problems with that in the future.